tests/system/test_requests.py

```diff
@@ -117,6 +117,9 @@ def test_deflate_error(self):
 class SecureTest(SecureServerBaseTest):
 
     def test_https_ok(self):
+        import urllib3
+
+        urllib3.disable_warnings()
```

Comment:
good finding!
Why is this a volatile error though?

Reply:
it started happening  on recent builds, not sure. maybe is caused by the beats merge...